src/generate_container_host_image.py

```diff
@@ -167,7 +167,7 @@ def get_slurm_gcp_docker_root():
 			except subprocess.CalledProcessError:
 				pass
 			subprocess.check_call(
-			  "gcloud compute images create {imagename} --source-snapshot={host}-snap --family {image_family}-$USER".format(imagename = imagename, host = host, image_family = args.image_family),
+			  "gcloud compute images create {imagename} --source-snapshot={host}-snap --family {image_family}-$USER-$UID".format(imagename = imagename, host = host, image_family = args.image_family),
```

Comment:
For some weird reason, `$UID` was blank when I tried running this:

```
Creating image from snapshot ...

ERROR: (gcloud.compute.images.create) Could not fetch resource:
 - Invalid value for field 'resource.family': 'slurm-gcp-docker-jhess-'. Must be a match of regex '(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)'

Traceback (most recent call last):
  File "./generate_container_host_image.py", line 169, in <module>
    subprocess.check_call(
  File "/usr/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'gcloud compute images create slurm-gcp-docker-v0-7-3-c4c1e0c-jhess --source-snapshot=dummyhost-jhess-snap --family slurm-gcp-docker-$USER-$UID' returned non-zero exit status 1
```

I'm not sure when we would encounter this edge case, since we are using OSLogin — the username is tied to the Google account handle. I suggest we just delete the `-$UID` suffix.

Reply:
I am mainly worrying about compatibility between remote-provisioned cluster image and local-provisioned cluster. However that won't be an issue if we explicitly state in the setup wiki that we need a VM with OS-login. I will revert the commit.